README.md

```diff
@@ -28,18 +28,20 @@ touch Brewfile
 Add your dependencies in your `Brewfile`:
 
 ```ruby
-cask_args appdir: "/Applications"
+tap "homebrew/core"
 tap "homebrew/cask"
-tap "telemachus/brew", "https://telemachus@bitbucket.org/telemachus/brew.git", pin: true
-tap "caskroom/fonts"
+tap "homebrew/cask-fonts"
+
+brew "emacs"
 brew "imagemagick"
 brew "mysql@5.6", restart_service: true, link: true, conflicts_with: ["mysql"]
-brew "emacs", args: ["with-cocoa", "with-gnutls"]
+
 cask "google-chrome"
 cask "java" unless system "/usr/libexec/java_home --failfast"
 cask "firefox", args: { appdir: "~/my-apps/Applications" }
+cask "font-charter"
```

Comment:
Use the full `"caskroom/fonts/font-charter"` format.

Reply:
Can this be “homebrew/cask-fonts/font-charter” instead?

Reply:
Yup!